.envrc

```diff
@@ -23,3 +23,8 @@ then
 fi
 
 export DBURL="postgres://postgres:postgres@localhost/autoscaler"
+
+if has pre-commit
+then
+  pre-commit install
+fi
```

Comment:
Don't get me wrong but I vote for completely opposite, which is about removing `pre-commit` from this project:
* smaller toolchain
* better dev-experience because the long running linter is not annoying the developer with every commit

Reply:
Yeah, my PR was a :trollface: - about removing `pre-commit` lets discuss in the team, maybe someone is still getting value from it,